src/main/java/com/Acrobot/ChestShop/Signs/ChestShopSign.java

```diff
@@ -11,16 +11,13 @@
 import com.Acrobot.ChestShop.UUIDs.NameManager;
 import com.Acrobot.ChestShop.Utils.uBlock;
 import org.bukkit.Bukkit;
-import org.bukkit.block.Block;
-import org.bukkit.block.BlockState;
-import org.bukkit.block.Chest;
-import org.bukkit.block.DoubleChest;
-import org.bukkit.block.Sign;
+import org.bukkit.block.*;
```

Comment:
This does also not need to be changed, as far as I can tell the previous imports are just fine for the added code?

Reply:
It doesnt _need_ to be changed, but I don't see what the problem is with optimised imports on a file that I actually do change.

Reply:
It modifies lines that are not required to be changed in the feature implemented by the commit. It therefore makes the git history annotation more difficult to read as it requires one more step to get the info on why those lines exist in the first place. Granted that doesn't play that big of a role on the imports but it's a good policy to keep the diff limited to the lines that actually matter to the feature.

Reply:
For actual code I completely agree with that policy and I apply it myself when possible, tho since I didn't even change the lines myself (plus the fact that its only imports) I don't see what the problem is. However its your repo and your plugin so I'll try my best to sort it out for you.